### DISCUSSION TITLE
lastfm artisti iamges

### DISCUSSION THREAD

**Discussion by cacko** [navidrome/navidrome#2436] [Ideas]
hi, can somebody implement that 

https://github.com/sentriz/gonic/commit/475749534f784eb803eec9cb83c72d49f30d1994

I just can't be bothered to learn that language sorry.. and if somebody can chunk that into navidrome, will be great.

idea is, get the artist, get the artist url page, and from that page, fetch the artist image from the opengraph headers

ta

**Comment by dweymouth** [comment]
Navidrome supports fetching artist images from Spotify if you have a Spotify account (free is fine) and create and associate a Spotify API key with Navidrome. I think it would be a good feature to add LastFM as an artist image source as well but just thought I'd mention the Spotify integration in case you weren't aware.

**Reply by dnno** [reply]
Substreamer supports fetching imagery from fanart.tv, that could be another Integration option.
